lib/buffer.js

```diff
@@ -14,11 +15,19 @@ Buffer.poolSize = 8 * 1024;
 var poolSize, poolOffset, allocPool;
 
 
+binding.setupBufferJS(Buffer.prototype, bindingObj);
+const flags = bindingObj.flags;
+const kNoZeroFill = 0;
+
+
 function createPool() {
   poolSize = Buffer.poolSize;
-  allocPool = binding.create(poolSize);
+  flags[kNoZeroFill] = 1;
+  allocPool = new Uint8Array(poolSize);
+  Object.setPrototypeOf(allocPool, Buffer.prototype);
```

Comment:
`class` is too slow. The call to `super()` can't be optimized by v8. I learned this b/c the initial implementation used `class...extends` and I never saw the constructor show up in IRHydra.